### ISSUE TITLE
Remove deprecated runtime dependency on libappindicator1 in Debian packages

### ISSUE THREAD

**Issue by RogueScholar** [wireapp/wire-desktop#6690]
Electron has finally managed to resolve this pain point as of [PR#36333](https://github.com/electron/electron/pull/36333) and now implements status (a/k/a system tray) icons via a desktop environment-agnostic DBus service instead of this old dinosaur of a library. It would be lovely to install a release of `wire-desktop` that didn't force its installation. As you can see on the [updated electron-builder documentation](https://www.electron.build/configuration/linux.html#debian-package-options) they acknowledge that it should be removed from the dependencies field to remain viable with Debian Stable and Debian-derivatives a la Ubuntu, Linux Mint, Pop_OS!, et cetera, et cetera.

If I may be so bold, I'd also like to lobby for following their guidelines regarding striking `libgconf-2-4` from the dependencies field as well. It's less of an aggravation than `lipappindicator1`, but it's something else that requires additional configuration for users of KDE Plasma to keep from interfering with the `plasma-workspace` system services and as it was only included for the system tray support, this would seem to be a good opportunity to yeet it in the spirit of packaging best practices and only listing dependencies needed for proper functionality. To that end, it might be worth examining if replacing `libnotify-bin` with `libnotify4` and adding `libxtst6` to align with the new packaging defaults wouldn't be wise.

I've not built Wire locally in awhile, but from what I remember of your toolchain I believe the relevant code to change would be with the enumerated elements of this array:
https://github.com/wireapp/wire-desktop/blob/8949dbef5d2827b2bc92aec693b4db2328b555cf/bin/build-tools/lib/build-linux.ts#L81

Thanks for making Wire and (hopefully) giving consideration to this request for changes in its packaging. 🙏🏻

**Comment by tlebon** [comment]
seems easy enough, i guess we will try to include this in the next release

**Comment by RogueScholar** [comment]
I was hoping that I'd brought this up with enough lead time to avoid major issues for end users, but now it appears that it wasn't. With the recent release of Ubuntu 23.10 "Mantic Minotaur", the `wire-desktop` package available on the Release page as well as the Wire APT repository is no longer installable due to package dependencies that are unable to be satisfied. The [`libgconf-2-4` package was withdrawn](https://tracker.debian.org/news/1451055/gconf-removed-from-testing/) from the Ubuntu Mantic package archives as well as from the Debian Testing and Debian Sid repositories on August 7th of this year. 

This compounds the original subject of this issue, which is that the `libappindicator1` package has been [absent from the Debian package repositories since September 14, 2021](https://tracker.debian.org/news/1257945/removed-0492-8-from-unstable/), removing it from the wider ecosystem of APT-based Linux distributions (with the exception of Ubuntu, who continues to maintain their own fork for the time being). I do hope that this relatively minor change can be incorporated into an upcoming release with haste now that the matter has escalated from an annoyance to a loss of ability to have a working installation. Thank you in advance for your consideration.

**Comment by buhund** [comment]
Having issues with this on (k)Ubuntu 24.04 deb-package.


```
The following packages have unmet dependencies:
 wire-desktop : Depends: libappindicator1 but it is not installable
                Depends: libgconf-2-4 but it is not installable
```

A workaround is removing the dependencies from the file specifying dependencies inside the .deb package, but this will make it unable to update properly, so one have to do the same workaround again each time one wants to update the package.

Please fix 😊

**Comment by RogueScholar** [comment]
Better late than never, haha. Fixed in #8240.

\*_cough, cough_\* Would be lovely if a new release was tagged and cut for distribution, too… \*_cough, cough_\*
